.changes/windows-http.md

```diff
@@ -0,0 +1,5 @@
+---
+"wry": "minor"
+---
+
+Add `WebViewBuilderExtWindows::with_http_scheme` to be able to choose between `http` and `https` for custom protocols on Windows. **Wry now uses `http` by default!**
```

Comment:
lets split this breaking change into its own change file, just so it is more clear.